sites/all/modules/custom/training/training.pages.inc

```diff
@@ -244,6 +244,10 @@ function training_theme_image_page(){
 function training_theme_access_page(){
   $access = t('Custom access for rendered content');
   $no_access = t('Custom access inversed for rendered content');
-  $text = user_access('custom access training') ? $access : $no_access;
-  return $text;
```

Comment:
In my comment I expected to see the following:
return user_access('custom access training') ? $access : $no_access;
Your code has no mistake. It is just for your information.